pkg/logql/log/parser_test.go

```diff
@@ -5,10 +5,9 @@ import (
 	"sort"
 	"testing"
 
+	"github.com/grafana/loki/pkg/logqlmodel"
 	"github.com/prometheus/prometheus/model/labels"
 	"github.com/stretchr/testify/require"
-
-	"github.com/grafana/loki/pkg/logqlmodel"
```

Comment:
Our linter requires that packages of the `github.com/grafana/loki` module are listed in a separate paragraph in third place.